src/engine/bank01.asm

```diff
@@ -5785,7 +5785,7 @@ Func_64fc: ; 64fc (1:64fc)
 	ld a, [wCurPlayAreaY]
 	inc a
 	ld e, a
-	ld d, $04
+	ld d, POKEMON_POWER
```

Comment:
This is actually used in InitTextPrinting_ProcessTextFromPointerToID for a text offset. So 4 is correct here.

Reply:
Did the same thing